api/api-common/src/main/kotlin/io/qalipsis/api/query/AggregationQueryExecutionContext.kt

```diff
@@ -30,6 +30,7 @@ import java.time.Instant
  * @property tenant the reference of the tenant owning the data
  * @property campaignsReferences references of all the campaigns that generated the data
  * @property scenariosNames names of all the scenarios that generated the data, defaults to empty for all the scenarios of the selected campaigns
+ * @property zones zones were the data was generated, defaults to empty for all the zones of the selected campaigns
```

Comment:
```suggestion
 * @property zones zones where the data was generated, defaults to empty for all the zones of the selected campaigns
```

This and others alike